components/collector/src/model/issue_status.py

```diff
@@ -43,12 +51,11 @@ def __init__(
     def as_dict(self) -> dict:
         """Return the issue status as dict."""
         status = dict(
-            name=self.name,
-            summary=self.summary,
+            status_category=self.status_category,
             parse_error=self.parse_error,
             connection_error=self.connection_error,
             api_url=self.api_url,
             landing_url=self.landing_url,
             **self.issue.as_dict(),
         )
-        return {key: value for key, value in status.items() if value}
+        return {key: value for key, value in status.items() if value is not None}
```

Comment:
Is it a problem that the nested Issue object is not displayed? Name and summary are now missing

Reply:
The nested issue object is included via the as_dict method.